lib/cc/engine/analyzers/ruby/main.rb

```diff
@@ -33,7 +34,37 @@ def overage(mass)
           end
 
           def process_file(file)
-            RubyParser.new.process(File.binread(file), file, TIMEOUT)
+            if File.extname(file) == ".erb"
+              process_erb file
+            else
+              RubyParser.new.process(File.binread(file), file, TIMEOUT)
+            end
+          end
+
+          def process_erb(file)
+            erb = File.read(file)
```

Comment:
This is a minor pondering that just occurred to me: we use `binread` above but just use `read` here. I suspect this is down to file encodings. You comfortable having these be different?